apps/datafeeder/src/app/presentation/pages/upload-data-page/upload-data.page.ts

```diff
@@ -16,7 +18,9 @@ marker('datafeeder.upload.error.subtitle.analysis')
 })
 export class UploadDataPageComponent implements OnInit {
   error: UploadDataError
-  maxFileSize = 30
+  get maxFileSizeMb() {
+    return parseSizeAsMb(SETTINGS.maxFileUploadSize)
```

Comment:
I don't see a test with this value mocked

Reply:
Right, I didn't test that part